### DISCUSSION TITLE
zed is not opening

### DISCUSSION THREAD

**Discussion by whois-root** [zed-industries/zed#9748] [Feature Requests]
Downloading zed for the Nth time. i thought reinstalling might solve the issue, but I don't know what's wrong. Somehow, i am not able to open zed now. It was running good, but then i tried to open a folder by dragging and dropping on the zed icon and after that it is not opening. I, then tried to run the zed unix exectuable file, and i managed to get some errors.

----------------------------------------------------------------
thread 'main' panicked at crates/zed/src/main.rs:472:54:
could not create tmp path: Os { code: 13, kind: PermissionDenied, message: "Permission denied" }
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.

[Process completed]
---------------------------------------------------------------

If anyone can help, please let me know what's the error here?

**Comment by Moshyfawn** [comment]
It sounds like the issue reported a couple of days ago: #9623.

**Reply by whois-root** [reply]
okay, i am a newbie. how can i open zed with sudo?

**Reply by the-jasoney** [reply]
`sudo zed [directory]`

**Reply by the-jasoney** [reply]
it will ask you for password

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
